ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/create-run.tsx

```diff
@@ -56,13 +57,25 @@ import {
   createRunFormSchema,
   CreateRunFormValues,
   defaultValues,
+  flattenErrors,
   formValuesToPayload,
 } from './-create-run-utils';
 
 const CreateRunPage = () => {
   const navigate = useNavigate();
   const params = Route.useParams();
   const ortRun = Route.useLoaderData();
+  const [openAccordions, setOpenAccordions] = useState<string[]>([]);
```

Comment:
Would this work everywhere if it was typed with accordion names like `("ananlyzer" | "scanner" ... )[]`

Reply:
Can you elaborate?

Reply:
```
  type AccordionSection =
    | 'analyzer'
    | 'advisor'
    | 'scanner'
    | 'evaluator'
    | 'reporter'
    | 'notifier';

  const [openAccordions, setOpenAccordions] = useState<AccordionSection[]>([]);

  // Manually toggle accordion open/close state
  const toggleAccordionOpen = (value: AccordionSection) => {
    setOpenAccordions(
      (prev) =>
        prev.includes(value)
          ? prev.filter((v) => v !== value) // Close accordion if open
          : [...prev, value] // Open accordion if closed
    );
  };
```

Reply:
Yeah exactly.